igor-web/src/main/java/com/netflix/spinnaker/igor/artifacts/ArtifactController.java

```diff
@@ -46,18 +46,20 @@ public ArtifactController(ArtifactServices artifactServices) {
   public List<String> getVersions(
       @PathVariable("provider") String provider,
       @PathVariable("name") String name,
-      @RequestParam(value = "releaseStatus", required = false) String releaseStatus) {
+      @RequestParam(value = "releaseStatus", required = false) List<String> releaseStatuses,
```

Comment:
This was previously limiting the ability to query artifacts matching multiple statuses.

Reply:
Is this change backwards compatible with the manual trigger dropdown only sending one status?

Reply:
Yes, because the way this is encoded in query parameters when it's a list is to have multiple occurrences of `releaseStatus=<status>` in the query string. This works for 1 to N.

Reply:
love it, so great!